internal/primitives/runtime/runtime.go

```diff
@@ -47,6 +47,17 @@ func (j Justifications) Get(engineID ConsensusEngineID) *EncodedJustification {
 	return nil
 }
 
+// Return a copy of the encoded justification for the given consensus
```

Comment:
```suggestion
// IntoJustification returns the encoded justification for the given consensus engine, if it exists.
```
(removing the mention of copying)